update ai-sanitizer doc (#8718)
Kong/docs.konghq.com: app/_hub/kong-inc/ai-sanitizer/overview/_index.md

## AI PII Anonymizer service

> Kong provides several [AI PII Anonymizer service](https://hub.docker.com/r/kong/ai-pii-service) Docker images, each of which has its own built-in NLP model and tagged with `version-lang_code` pattern. For example `ai-pii-service:v0.1.2-en` means the version of the image is 0.1.2 and its built-in NLP model is English, `ai-pii-service:v0.1.2-it` is for Italian and `ai-pii-service:v0.1.2-fr` is French, etc. The image tagged with `all` contains English, French, German, Italian, Japanese, Portuguese and Spanish models. For the requirement to use other NLP models, please customize `ai_pii_service/nlp_engine_conf.yml` with any one of the available images.

Reviewer: ```suggestion
Kong provides several [AI PII Anonymizer service](https://hub.docker.com/r/kong/ai-pii-service) Docker images, each of which has its own built-in NLP model and is tagged with the `version-lang_code` pattern. For example:
* `ai-pii-service:v0.1.2-en` means the version of the image is 0.1.2 and its built-in NLP model is English
* `ai-pii-service:v0.1.2-it` is for Italian and `ai-pii-service:v0.1.2-fr` is French
and so on.

The image tagged with `all` contains English, French, German, Italian, Japanese, Portuguese, and Spanish models. If you need to use other NLP models, customize `ai_pii_service/nlp_engine_conf.yml` with any of the available images.
```
